process-engine/correlation.service
# ✨ Add claim check for super admin mode

## Changes

1. Add a claim check for the `can_manage_process_instances` claim, which indicates a super admin. 
    - These types of admins are always allowed to see any correlation and will therefore always get a pass during correlation filtering.
2. Replace TSLint with ESLint
3. Replace Gulp with a purely TSC based build
4. Apply new ESLint styleguide
5. Remove unhelpful code docs
6. Allow linter errors to cause Jenkins build failures

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/336

PR: #6

## How to test the changes

Best use BPMN Studio and our [IdentityServer](https://github.com/process-engine/identity_server)

- Start a ProcessInstance as User `alice` (has the super admin claim)
- Start a ProcessInstance as User `bob` (is a normal user)
- Login as `alice` and look at the Correlation list
    - The user can see two process instances
- Login as `bob` and look at the Correlation list
    - The user can only see his own process instance
